src/main/resources/static/js/status-handler.js

```diff
@@ -0,0 +1,159 @@
+/*
+This js file intend to keep the status information about the current selected back-end
+and depending weather the back end status has changed and or is online or not will
+enable or disable different kinds of status icons/fields.
+It may also fore page refresh pages, remove subscriptions from subscription list
+and reload subscriptions in subscription list depending on the back-end status.
+*/
+var pagesForStatusIndication = "subscriptions test-rules ei-info login";
+
+var statusType = {
+    success: "alert-success",
+    info: "alert-info",
+    warning: "alert-warning",
+    danger: "alert-danger"
+};
+
+var statusText = {
+    backend_down: "<strong>Back end is down!</strong> Wait for it go up or switch to another back end before continuing!",
+    test_rules_disabled: "<strong>Test Rule service is disabled!</strong> To enable it set the backend property [testaggregated.enabled] as [true]"
+};
+
+var backEndStatus = true;
+var previousBackEndStatus;
```

Comment:
I don't think you should change them to backEnd(from backend). Didn't we agree that we will merge the words as it is now accepted as an technical term just like "e-mail -> email", "plug-in" -> plugin etc.

Reply:
In documentation all got changed to back-end.

Reply:
Then why do you call it "back end" here then? With that logic you could as well use the name that is more commonly used as a technical term "backend". :) I won't put -1 because of this, it's up to you. I've just given my opinion.